index.html

```diff
@@ -217,6 +210,50 @@ <h2 class="about-me-title">About me</h2>
     </div>
   </section>
 
+  <section id="contact" class="contact-form-container contact-form-desktop">
+    <div class="contact-container">
+      <p class="support-text-form-section">I'm always interested in hearing about new
+        projects, so if you'd like to chat please get in touch.
+      </p>
+      <form class="contact-form-mobile contact-form-desktop" action="https://formspree.io/f/mlekejwj" method="post">
+        <input type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required />
+        <hr class="separator-line">
+        <input type="email" id="mail" name="user_email" placeholder="Email address" required />
+        <hr class="separator-line">
+        <textarea class="textarea_mobile" id="msg" name="user_message" maxlength="500" placeholder="Enter text here"
+          required></textarea>
+        <input class="button-colors" type="submit" value="Get in touch"></input>
+      </form>
```

Comment:
**1-** Please consider making the form elements in the center horizontally to make it look alike the Figma Template.

## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/228649721-2778e68c-8412-4c71-b794-7a956c8bbd80.png)


## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/228649782-e881a63b-be31-4415-97c7-e61497dac60e.png)

**2-** Moreover, please consider adding or removing the outline from all the input including the text area field to match the design. The outline comes when the user clicks on the input.

```
outline: none;
```